examples/apps/cra-demo/README.md

```diff
@@ -156,23 +159,25 @@ function useKVPair(): [KVData, SetKVPair | undefined] {
 
     }, [])
 
-    return [data, dataObject?.set];
+    const setPair = dataObject?.set;
+
+    return [data, setPair];
 }
 ```
 
 ## 7. Syncing our app state with Fluid data
 
-It is possible to avoid syncing data between Fluid and app state, but I've found that it can cause as many problems as it solves. So for this demo we will have app state drive our UI updates, and sync Fluid data into our view's state anytime that it changes. The advantages of this approach are:
+It is possible to avoid syncing data between Fluid and app state, but I've found that it can cause as many problems as it solves. So for this demo we will have app state drive our UI updates, and sync Fluid data into our view's state any time that it changes. The advantages of this approach are:
```

Comment:
For a future update - we should call out the _dis_ advantages of this approach as well, like increased (double?) memory usage.

Reply:
yeah, i don't love this section as it really glosses over things, could certainly use another pass